imgee/storage.py

```diff
@@ -150,7 +171,11 @@ def get_fitting_size((orig_w, orig_h), size):
     >>> get_fitting_size((200, 500), (400, 600))
     [240, 600]
     """
-    if size[0] == 0 and size[1] == 0:
+    if orig_w == 0 or orig_h == 0:
+        # this is either a cdr file or a zero width file
```

Comment:
What's a "zero width" file? Technically that can't be an image. Helps to clarify which of the following is the case:

1. We are unable to parse this image to extract its size.
2. It's a vector image with no size in pixels. However, it will still have an aspect ratio that matters when resizing.